openedx/core/djangoapps/course_roles/data.py

```diff
@@ -0,0 +1,160 @@
+"""
+Permissions for course roles app.
+
+These are the permissions that can be assigned to a CourseRole which grants access for a course,
+but can also be assigned for org wide course access or instance wide course access.
+They are defined in the database in the course_roles_permission table.
```

Comment:
Nit: You might want to remind folks reading this that edX has blue/green deployments, so they're going to be running in a state where some of the code has the new permissions and some won't. So old code needs to be resilient to seeing new permissions it doesn't recognize.